components/script/dom/bindings/codegen/CodegenRust.py

```diff
@@ -4475,13 +4474,13 @@ def getBody(self):
             attrs = "JSPROP_ENUMERATE"
             if self.descriptor.operations['IndexedSetter'] is None:
                 attrs += " | JSPROP_READONLY"
-            fillDescriptor = ("desc.get().value = result_root.ptr;\n"
```

Comment:
This is a behaviour change, is it not? Can we avoid those in this PR? Keeping the buggy code with a TODO comment until such time we can write a test for it seems better to me.